src/pngwriter.cc

```diff
@@ -2023,6 +2023,11 @@ void pngwriter::write_png(void)
 ///////////////////////////////////////////
 void pngwriter::plot_text( char * face_path, int fontsize, int x_start, int y_start, double angle, char * text, double red, double green, double blue)
```

Comment:
what we [actually want](https://stackoverflow.com/questions/890535/what-is-the-difference-between-char-const-and-const-char) is a `const char * const` or even better a c++ `const std::string`